.github/workflows/build_ghcr.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         ubuntu: [jammy]
         platform: [linux/amd64]
+        cycle: [devel, release]
```

Comment:
Can we use the convention `branch` rather than `cycle`?

In the context of the salt formulas, I use `cycle` with the possible values `patch` or `devel`, which is super confusing but I was trying to follow an R convention/lingo.  I just use it to determine if `BiocManager::install(version="devel")` should be run, which is during the period of time when we use the same version of R on release and devel.